docs/3.0rc/deploy/serve-flows/deploy-a-flow.mdx

```diff
@@ -312,10 +312,10 @@ executing. Pass `timeout=0` to return immediately and not block.
     
 If you call `run_deployment` from within a flow or task, the scheduled flow
 run is linked to the calling flow run (or the calling task's flow run)
-as a subflow run by default.
+as a nested flow run by default.
 
-Subflow runs have different behavior than regular flow runs. For example, you can't suspend a 
-subflow run independently of its parent flow. To not link the scheduled flow run to the 
+Nested flow runs have different behavior than regular flow runs. For example, you can't suspend a 
+nested flow run independently of its parent flow. To not link the scheduled flow run to the 
```

Comment:
```suggestion
nested flow run independently of its parent flow run. 
To prevent the flow run triggered by `run_deployment` from being linked to the flow run from which the method was called,
```